### PR TITLE
Add Apple Juice

### PR DESCRIPTION
**Pull request by jelflein**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating only the `sha256`**:

- [x] I verified this change is legitimate [<sup>How do I do that?</sup>][version-checksum] and **am providing confirmation below**:

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/apple-juice.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'apple-juice' do
+  version '1.2.3.1'
+  sha256 'a30c9d271bcda8d48c670dbf7e98c2a2c955152c413caea0868889d56b15e258'
+
+  url "https://github.com/raphaelhanneken/apple-juice/releases/download/#{version}/Apple.Juice.dmg"
```
Comment (commitay) [inline Casks/apple-juice.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)

Example:
https://github.com/caskroom/homebrew-cask/blob/0480e1f119f2742586f35b65a4efbfba670757b8/Casks/autodmg.rb#L5-L8

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/apple-juice.rb (hunk 1 above)
Comment (jelflein) [reply Casks/apple-juice.rb]: I added appcast / checkpoint.

Comment (commitay) [reply Casks/apple-juice.rb]: Thanks!

**Comment by commitay** [comment]
Needs `uninstall`

**Review by commitay — APPROVED** [review]

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**